WordPress/Classes/Utility/BuildInformation/FeatureFlag.swift

```diff
@@ -38,8 +38,8 @@ enum FeatureFlag: Int, CaseIterable {
             return false
         case .compliancePopover:
             return true
-        case .domainFocus:
-            return true
+        case .googleDomainsCard:
+            return false
```

Comment:
Optional:

I think we could add a comment here saying that the functionality will be removed with a follow up pr and/or link the draft PR. If you think that it isn't necessary as you are already working on it, that's also fine.